sections/footer.liquid

```diff
@@ -128,14 +107,14 @@
                   <label class="field__label" for="NewsletterForm--{{ section.id }}">
                     {{ 'newsletter.label' | t }}
                   </label>
+                  <button type="submit" class="newsletter-form__button field__button" name="commit" id="Subscribe" aria-label="{{ 'newsletter.button_label' | t }}">
```

Comment:
The button and button text is now an arrow. My very light [research](https://splitbase.com/blog/luxury-ecommerce-crm) had negative findings on that:

- See Prada example, they use arrow and have negative usability “review”:  doesn’t look like a button and can be hard to click. Second, it doesn’t direct an action. Great call to action buttons have text that incites the visitor to complete the action (e.g. “Sign Up”). Here we’ve got nothing.
- See Gucci example, another arrow: This is a usability flaw, but also a communication flaw. It’s not clear that by clicking on it you’re signing up, and since there’s no text on that button, it’s not “pushing” anyone to take action. 

Hopefully there is other, positive research.

Was the mail/envelope icon discussed instead of an arrow? There are a few examples of that and maybe that’s 10% clearer, but still less clear than a button with 1-3 words, e.g.  was `Subscribe` but accessibility concerns that this may not stand out.

**And finally to the copy to help end-user customers subscribe:** 
Can we add an explicit label on hover and/or aria label?
`Click to subscribe to emails`

- Shopify allows `Click`, at least internally: Example: These emails are automatically sent out to either you or the customer. Click on the email templates below to edit.
- Some UX writing practice objects to this b/c of accessibility and uses `Select` to cover more actions

Reply:
Worth noting, this pattern has also been implemented on at least the search page https://screenshot.click/09-18-cgim2-v95j9.png, but it does use a more informative icon than an arrow. Is a mail/envelope icon a reasonable enough solution here @svinkle or is the lack of a visible button label a dealbreaker in general?

Reply:
@kmeleta There could be issues here, too. I'm curious if this pattern has gone through usability testing, or if there are plans for this? I don't want my comment to be a bottleneck for this PR, so let's leave as-is for now.

@emma-boardman Are we in a good spot to revisit testing with Fable? I believe last time we chatted about include more than screen reader users.

Reply:
> @emma-boardman Are we in a good spot to revisit testing with Fable? I believe last time we chatted about include more than screen reader users.

I'd be delighted to moderate if the Dawn team are interested in doing another round 👍